docs/docs/how-to/performance/improving-site-performance.md

```diff
@@ -45,13 +47,13 @@ Performance work can be surprisingly nonlinear in impact.
 
 ## Part 2: Implement Improvements.
 
-When you run a test in your testing tool of choice, it will give you a number of recommendations. While this can feel like a laundry list of issues, it can be helpful to understand the five core categories that these issues are bucketed into.
+When you run a test in your testing tool of choice, it will give you a number of recommendations. These can be bucketed into five core categories:
 
-- Blocking calls & third-party scripts.
-- JavaScript bundle size.
-- Stylesheets and font files.
-- Images and other media.
-- Resource requests & CDN caching configuration.
+- [Blocking calls & third-party scripts](#address-third-party-script-impact).
+- [Javascript bundle size](#reduce-your-javascript-bundle-cost).
```

Comment:
The correct name is `JavaScript` not `Javascript` (throughout the document)

Reply:
Bad habit of mine, I do this with WordPress too! Thanks :)